Documentation/guides.xml

```diff
@@ -9,6 +9,8 @@
         class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension"
         edit-on-github="TYPO3-Documentation/render-guides"
         edit-on-github-branch="main"
+        official-manual-interlink-shortcode="t3docsRenderGuides"
+        official-manual-interlink-predefined="true"
```

Comment:
Why this predefined? What's the consequences?

Reply:
removed for now, we might not need it